locales/zh_CN/LC_MESSAGES/export_results.po

```diff
@@ -288,46 +338,48 @@ msgstr ""
 
 #: ../../export_results.rst:150
 msgid "Install ``ffmpeg`` on CentOS and RHEL"
-msgstr ""
+msgstr "在  CentOS 和 RHEL 上安装 ``ffmpeg``"
 
 #: ../../export_results.rst:152
 msgid "sudo yum install ffmpeg ffmpeg-devel"
 msgstr ""
 
 #: ../../export_results.rst:156
 msgid "Install ``ffmpeg`` on Arch Linux:"
-msgstr ""
+msgstr "在 Arch Linux 上安装 ``ffmpeg``:"
 
 #: ../../export_results.rst:162
 msgid "Test your installation using"
-msgstr ""
+msgstr "使用下面这行命令测试你的安装是否成功"
 
 #: ../../export_results.rst:164
 msgid "ffmpeg -h"
 msgstr ""
 
 #: ../../export_results.rst:169
 msgid "Install ``ffmpeg`` on OS X"
-msgstr ""
+msgstr "在 OS X 上安装 ``ffmpeg``"
 
 #: ../../export_results.rst:171
 msgid "``ffmpeg`` can be installed on OS X using ``homebrew``:"
-msgstr ""
+msgstr "在  OS X  上可以通过 ``homebrew`` 安装 ``ffmpeg``:"
 
 #: ../../export_results.rst:173
 msgid "brew install ffmpeg"
 msgstr ""
```

Comment:
```suggestion
msgstr "使用 brew 安装 ``ffmpeg``"
```
Not translated? :-)）

Reply:
This is the command line to install ``ffmpeg`` on OS X. Keeping origin text will make it easier for readers to use ,I think 🙆‍♂️

Reply:
opps my bad. Thought it's the headline. Revert.